dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/impl/ResourcesServiceImpl.java

```diff
@@ -847,7 +847,12 @@ private boolean upload(User loginUser, String fullName, MultipartFile file, Reso
     @Override
     public Map<String, Object> queryResourceList(User loginUser, ResourceType type, String fullName) {
         Map<String, Object> result = new HashMap<>();
-
+        if (!PropertyUtils.getResUploadStartupState()) {
+            logger.error("Storage does not start up, resource upload startup state: {}.",
+                    PropertyUtils.getResUploadStartupState());
```

Comment:
The error message should tell users like 'You need to set the resource storage in the file of common.properties.'